aws/resource_aws_lambda_function.go

```diff
@@ -105,6 +105,15 @@ func resourceAwsLambdaFunction() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"layers": {
+				Type:     schema.TypeSet,
```

Comment:
I think the ordering of layers matters according to the AWS documentation: https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html

> Layers are applied in the order that's specified, merging any folders with the same name. If the same file appears in multiple layers, the version in the last applied layer is used.

Given this, we should switch this to `TypeList` here and in the data source. 👍

Reply:
Gotcha - will do

Reply:
done!